lib/Doctrine/Migrations/Version/DbalExecutor.php

```diff
@@ -301,23 +302,24 @@ private function executeResult(MigratorConfiguration $configuration) : void
                 continue;
             }
 
-            $this->logger->debug('{duration}ms', [
+            $this->logger->notice('Query took {duration}ms', [
                 'duration' => $stopwatchEvent->getDuration(),
             ]);
         }
     }
 
-    private function outputSqlQuery(Query $query) : void
+    private function outputSqlQuery(Query $query, MigratorConfiguration $configuration) : void
     {
         $params = $this->parameterFormatter->formatParameters(
             $query->getParameters(),
             $query->getTypes()
         );
 
-        $this->logger->debug('{query} {params}', [
-            'query' => $query->getStatement(),
-            'params' => $params,
-        ]);
+        $this->logger->log($configuration->getTimeAllQueries() ? LogLevel::NOTICE : LogLevel::DEBUG,
+            '{query} {params}', [
+                'query'  => $query->getStatement(),
+                'params' => $params,
+            ]);
```

Comment:
This is a PSR-2 violation, the call has to be fully multiline.

Reply:
Guess I fixed it now 😃